normalized-cache-incubating/src/commonMain/kotlin/com/apollographql/cache/normalized/FetchPolicyInterceptors.kt

```diff
@@ -130,14 +131,14 @@ val CacheAndNetworkInterceptor = object : ApolloInterceptor {
 }
 
 /**
- * If this response is a cache miss, returns a response with an exception, otherwise returns this response.
+ * If this response has errors, returns a response with an exception, otherwise returns this response.
  * This can be used to accommodate [com.apollographql.apollo.ApolloCall.execute] which splits responses based on exceptions.
```

Comment:
```suggestion
 * If this response has errors, returns a response with an exception, otherwise returns this response.
 * This can be used to accommodate [com.apollographql.apollo.ApolloCall.execute] which splits responses based on exceptions and should only be called on cache responses.
```